lib/utils/ast.js

```diff
@@ -49,6 +51,84 @@ function isBinaryExpression(node) {
   return node && node.type === 'BinaryExpression';
 }
 
+function isCallExpression(node) {
+  return node && node.type === 'CallExpression';
+}
+
+function isMemberExpression(node) {
+  return node && node.type === 'MemberExpression';
+}
+
+function isIdentifier(node, name) {
+  return node 
+  && node.type === 'Identifier' 
+  && (name === undefined || node.name === name);
+}
+
+function isDescribeAlias(node) {
+  return isIdentifier(node) && node.name === 'describe';
+}
+
+function isAccessor(node, value) {
+  return isIdentifier(node, value) || isStringNode(node, value);
+}
+
+function isStringNode(node, specifics) {
+  return isStringLiteral(node, specifics) || isTemplateLiteral(node, specifics);
+}
+
+function isTemplateLiteral(node, value) {
+  return node.type === 'TemplateLiteral'
+  && node.quasis.length === 1
+  && (value === undefined || node.quasis[0].value.raw === value);
+}
+
+function getStringValue(node) {
```

Comment:
Why all this extra stuff with template literals and accessor values?

Reply:
Chalk it up to ignorance. I heavily used `eslint-plugin-jest`'s implementation of this rule as a guideline, and this being my first time really working with the JavaScript AST, I assumed that this logic was covering some kind of edge case I didn't fully understand. You know what they say about assuming things...

I'll go ahead and simplify this.